content/en/community/_index.md

```diff
@@ -71,156 +62,4 @@ To participate, fill out the [Kyverno adopters form](https://forms.gle/K5CApcBAD
-- A maintainer has made less than 30 contributions over a span of 6 months.
-  - Contributions can be tracked using the [DevStats dashboard](https://kyverno.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%206%20months&var-metric=contributions&var-repogroup_name=All&var-country_name=All&from=1522810884223&to=1680577284223&var-companies=All).
-  - Other relevant data will be collected and evaluated to assess the maintainer's contributions. This includes their involvement in discussions, conversations on Slack, and any other relevant interactions.
-
-The off-boarding process includes the following steps:
-- The off-boarding process is initiated by any currently active maintainer who conducts a review of the maintainers list and proceeds to initialize the off-boarding process if the above criteria are met.
-- The plans of off-boarding process is sent in a private Slack message or email to the candidate.
-- If the candidate for removal states plans to continue participating, another 6 months will be granted to the candidate to make contributions and the new cycle starts. No action is taken and this process terminates.
-- If the candidate fails to meet the criteria during the second attempt to make contributions, the off-boarding process continues.
-- A pull request (PR) proposing movement of the candidate is sent, initiating the public voting phase.
-- The vote passes if a majority of current maintainers vote yes during a voting period lasting seven (7) days.
-- A positive vote will result in movement to an _emeritus_ status within `MAINTAINERS.md` and removal from organization membership.
-
-#### Admins
-
-These are persons who have full access to the project, including sensitive and destructive actions like managing security or deleting a repository. Admins can read, clone, and push to this repository. They can also manage issues, pull requests, and repository settings, including adding collaborators.
-
-#### Mapping Project Roles to GitHub Roles
-
-The roles used in this document are custom roles mapped according to the [GitHub roles and responsibilities](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization).
-
-| Project Role   | GitHub Role    |
-| -------------- | -------------- |
-| Contributor    | Triage         |
-| Code Owner     | Write          |
-| Maintainer     | Maintain       |
-| Administrator  | Admin          |
-
-### Off-boarding Guidance
-
-If any of the above roles hasn't contributed in any phases (including, but not limited to: code changes, doc updates, issue discussions) in 3 months, the administrator needs to inform the member and remove one's roles and GitHub permissions.
+[Kyverno and its sub-projects](https://github.com/kyverno#projects) follow the governance published and maintained at https://github.com/kyverno/community/blob/main/GOVERNANCE.md.
```

Comment:
The `Roles` in https://github.com/kyverno/community/blob/main/GOVERNANCE.md point back to this:

https://github.com/kyverno/community/blob/main/GOVERNANCE.md#roles

Reply:
Added in https://github.com/kyverno/community/pull/7.